asserter/operation_status.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Coinbase, Inc.
```

Comment:
2024？？？

Reply:
from what I saw, we have never changed copyright in asserter last 4 years. only those files in types, client, and server directories have been changed every year. 

but anyway, I just changed all files which have copyright 2020.